$OpenBSD: patch-app_main_c,v 1.1 2011/04/16 07:47:04 dcoppa Exp $

Using G_OPTION_FLAG_NO_ARG with G_OPTION_ARG_NONE is not a valid
combination and glib 2.28.2 warns about it.

http://git.gnome.org/browse/gimp/commit/?id=b81276ca83e6ec4c7df32c8f611f441d1dfc59ad

--- app/main.c.orig	Fri Apr 15 20:18:24 2011
+++ app/main.c	Fri Apr 15 20:19:15 2011
@@ -240,7 +240,7 @@ static const GOptionEntry main_entries[] =
     N_("Debug in case of a crash (never|query|always)"), "<mode>"
   },
   {
-    "debug-handlers", 0, G_OPTION_FLAG_NO_ARG,
+    "debug-handlers", 0, 0,
     G_OPTION_ARG_NONE, &use_debug_handler,
     N_("Enable non-fatal debugging signal handlers"), NULL
   },
